src/libraries/Microsoft.Extensions.DependencyInjection/tests/DI.External.Tests/Autofac.cs

```diff
@@ -9,8 +9,6 @@ namespace Microsoft.Extensions.DependencyInjection.Specification
 {
     public class AutofacDependencyInjectionSpecificationTests : SkippableDependencyInjectionSpecificationTests
     {
-        public override bool SupportsIServiceProviderIsService => false;
```

Comment:
Can we get rid of the `SupportsIServiceProviderIsService` boolean all together now that all the External DI implementations support it?

Reply:
no because `Lamar` is broken for example